### DISCUSSION TITLE
Variable length sequences

### DISCUSSION THREAD

**Discussion by 94Sip** [alan-turing-institute/sktime#1291] [Q&A]
I see that there is a PaddingTransformer for dealing with variable length sequences. In the context of time series classification, are each of the various algorithms capable of handling these masked/padded values, so they don't take those values into account?

Thank you.

**Comment by fkiraly** [comment]
no, all algorithms will handle them as if the sequences were equal length after the padding.

Most sktime algorithms are *not* capable handling unequal length time series, and the padding transformer can be used to still have some way of applying them to unequal length.
